src/biokbase/narrative/tests/test_jobcomm.py

```diff
@@ -148,190 +169,231 @@ def setUp(self):
         self.jc.stop_job_status_loop()
         self.job_states = get_test_job_states()
 
-    def check_error_message(self, req, source, input_, err):
+    def check_error_message(self, req, err, extra_params={}):
```

Comment:
simplified this test helper as having the raw request means numerous error fields can be removed